diff -urN SYMMLQ.orig/configure.in SYMMLQ.patch/configure.in
--- SYMMLQ.orig/configure.in	1970-01-01 01:00:00.000000000 +0100
+++ SYMMLQ.patch/configure.in	2004-12-16 12:04:25.922290400 +0100
@@ -0,0 +1,11 @@
+dnl Process this file with autoconf to produce a configure script.
+AC_INIT([symmlq], [1.0.0])
+AM_INIT_AUTOMAKE
+AM_CONFIG_HEADER([config.h])
+
+dnl Checks for programs.
+AC_ISC_POSIX
+AC_PROG_F77
+AM_PROG_LIBTOOL
+
+AC_OUTPUT([Makefile])
diff -urN SYMMLQ.orig/SRC/Makefile.am SYMMLQ.patch/SRC/Makefile.am
--- SYMMLQ.orig/Makefile.am	1970-01-01 01:00:00.000000000 +0100
+++ SYMMLQ.patch/Makefile.am	2004-12-16 12:08:47.655500904 +0100
@@ -0,0 +1,5 @@
+LIBTOOL = $(SHELL) $(top_builddir)/libtool $(LTTAG)
+lib_LTLIBRARIES = libsymmlq.la
+libsymmlq_la_LDFLAGS = -version-info $(subst .,:,@VERSION@)
+libsymmlq_la_LIBADD =$(LDFLAGS)
+libsymmlq_la_SOURCES= symmlq.f
